test/acceptance/features/steps/openshift.py

```diff
@@ -334,3 +335,25 @@ def search_resource_lst_in_namespace(self, resource_plural, name_pattern, namesp
             return lst
         print('Resource list is empty under namespace - {}'.format(namespace))
         return None
+
+    def get_secret_data(self, secret_name, secret_key, namespace, secret_value):
+        json_path = f'.data.{secret_key}'
+        if secret_value == "true":
+            start = 0
+            timeout = 180
+            interval = 5
+            while (start <= timeout):
+                output = self.get_resource_info_by_jsonpath("secrets", secret_name, namespace, "{" + json_path + "}", wait=True, timeout=30)
```

Comment:
If you modify json_path defination as `json_path = f'{.data.{secret_key}}'` and remove curely  `"{" + json_path + "}"` then this line will look more cleaner `self.get_resource_info_by_jsonpath("secrets", secret_name, namespace, json_path, wait=True, timeout=30)`

Reply:
Mea culpa! you'll need to use extra braces around `json_path = f'{{.data.{secret_key}}}'`, I think this should work.

Reply:
ERROR MESSGE: b"error: error parsing jsonpath {{.data.CustomReady}}, unrecognized character in action: U+007B '{'\n"
      ERROR CODE: 1

Reply:
Doesn't work. I had tried that earlier too

Reply:
it looks like you have some new line - try to trim it.

Reply:
I am keeping the "{" with the parameters as it works for now. If it passes for you please feel free to send a PR against my branch :)

Reply:
It should work, I tried it with your PR on my system.

Reply:
works now, thanks @pratikjagrut . pushing changes and resolving this

Reply:
https://github.com/redhat-developer/service-binding-operator/pull/597/commits/f0136cc8565b4aa248f5dafc58922680e88edd03